components/p3a/buildflags.gni

```diff
@@ -1,5 +1,5 @@
 import("//build/config/features.gni")
 
 declare_args() {
-  brave_p3a_enabled = !is_ios && !is_android
+  brave_p3a_enabled = !is_ios
```

Comment:
AFAIU, `is_ios` is redundant, because we don't build p3a on ios anyway. So I suggest to drop the whole buildflag

Reply:
I didn't know that, I know that a lot of brave-core code is used on iOS, especially for rewards and sync now.

if we drop `!is_ios` , I would like to put the comment that p3a is not used for iOS,  like
```
# for iOS P3A is not used
brave_p3a_enabled = true
```
so to keep the existing for me seems good because it is self-described line

Reply:
AFAIK generally we don't use `is_ios` since it wouldn't work anyway, only selected components you mention support that. So I think we should be safe to drop the flag - @bsclifton could you help us?

Reply:
iOS guys told it should be safe. I will remove that guard and run CI.

Reply:
CI for iOS succeeded with that change https://github.com/brave/brave-core/pull/7016/commits/95553fa5a073d33ffe9710ef76926d36d57275af